.github/workflows/release.yml

```diff
@@ -6,6 +6,10 @@ on:
       tag:
         description: Tag for release
         required: true
+      draft:
+        description: draf release will not make the final image and skip some particular steps
```

Comment:
`darf release`? 😅 

`skip some particular steps` is vauge. Let’s avoid wordings like this and explain the most important things here.

Is it `will not build and publish a Docker image on Dockerhub`?